app/store/services/clubs.js

```diff
@@ -4,6 +4,9 @@ import { pick } from "lodash";
 import { API_ENDPOINT, RESOURCES, METHODS } from "../constants/rest";
 import { callJSONApi } from "./helpers/api";
 
+const computeID = (min, max) =>
+  Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
wtf is that +1

Reply:
random shit function I needed due to strava breaking api change! 
+1 is my legacy 🌮